### ISSUE TITLE
There have panic when run `lighthouse` because of csp_evaluator@1.0.2 is incompatible

### ISSUE THREAD

**Issue by wqcstrong** [GoogleChrome/lighthouse#12856]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

[Just install lighthouse and run will display]

### What happened?

The `lighthouse` dependens on csp_evaluator@^1.0.1 and panic in csp_evaluator@1.0.2

### What did you expect?

Run `lighthouse` normally

### What have you tried?

Install `lighthouse` locally instead of global, and install specified version via `yarn upgrad csp_evaluator@1.0.1`, everything be ok after that.

### How were you running Lighthouse?

CLI

### Lighthouse Version

lighthouse@8.2.0

### Chrome Version

/

### Node Version

/

### Relevant log output

```shell
/
```

**Comment by dazulu** [comment]
Probably explains why I am getting this today
<img width="944" alt="Screenshot 2021-08-04 at 13 25 11" src="https://user-images.githubusercontent.com/4543447/128174632-c0a9b9c1-4392-4e30-b1e3-6834731c7fd1.png">

**Comment by adamraine** [comment]
https://github.com/google/csp-evaluator/issues/36

**Comment by javigomez** [comment]
I am also getting this.
```
Cannot find module 'csp_evaluator/dist/lighthouse/lighthouse_checks.js'\nRequire stack:\n- /node_modules/lighthouse/lighthouse-core/lib/csp-evaluator.js\n- /node_modules/lighthouse/lighthouse-core/audits/csp-xss.js\n- /node_modules/lighthouse/lighthouse-core/config/config-helpers.js\n- /node_modules/lighthouse/lighthouse-core/config/config.js\n- /node_modules/lighthouse/lighthouse-core/index.js\n- /src/domain/lighthouse-report.js\n- /src/domain/lighthouse-reports-manager.js\n- /src/infrastructure/cli-lighthouse.js","timestamp":"2021-08-04T15:24:41.454Z"}}
```

It all started  between 11am and 12am GMT today

**Comment by adamraine** [comment]
This *should* be resolved with the release of `csp_evaluator` 1.0.4

**Comment by javigomez** [comment]
I can confirm the issue is resolved
